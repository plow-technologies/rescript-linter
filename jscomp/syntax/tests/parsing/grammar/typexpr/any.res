type t = _

let t: _ = x
