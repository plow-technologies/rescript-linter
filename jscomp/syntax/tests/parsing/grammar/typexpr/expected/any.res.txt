type nonrec t = _
let (t : _) = x